util/test_timeline.py

```diff
@@ -236,11 +262,7 @@ def test_match(event, entry):
         elif entry["special_type"] == "battlelog" and event.startswith(entry["special_line"]):
             # Matching this format generically:
             # 00|2019-01-12T18:08:14.0000000-05:00|0839||The Realm of the Machinists will be sealed off in 15 seconds!|
-            log_match = re.search(
-                "^00\|[^\|]*\|{}\|[^\|]*\|{}".format(entry["logid"], entry["line"]),
-                event,
-                re.IGNORECASE,
-            )
+            log_match = re.search(entry["regex"], event, re.IGNORECASE,)
```

Comment:
*This* little annoyance right here was what made `gainsEffect` syncs perpetually fail. We can't rely on the casing of the `effectId` in the timeline file to match that in the network log. ~~It took me "only" half an hour of "WHY WON'T THIS WORK? IT MATCHES!" to debug~~